android/build.gradle

```diff
@@ -133,8 +143,8 @@ dependencies {
 
 if (isNewArchitectureEnabled()) {
```

Comment:
My goal was to treat the new architecture as opt-in. Maybe removing this check here does not cause any trouble, but I'd keep it for safety.

Reply:
Sorry, what I meant is that since RN `0.70`, all those lines are not needed: https://github.com/Kureev/react-native-blur/pull/493/files/1af1251593436cff9015115886fdfd0edd025268#diff-197b190e4a3512994d2cebed8aff5479ff88e136b8cc7a4b148ec9c3945bd65aR144-R149.

Reply:
Lines removed; I moved `javaPackageName` to Codegen config in `package.json`.